extensions/amp-story-panning-media/0.1/amp-story-panning-media.js

```diff
@@ -40,27 +43,30 @@ export class AmpStoryPanningMedia extends AMP.BaseElement {
     /** @private {?Element} The element that is transitioned. */
     this.ampImgEl_ = null;
 
-    /** @private {?string} Sent to siblings to update their position. */
+    /** @private {?number} */
```

Comment:
nit: can you comment for `this.x_` and `this.y_` what the expected unit is?  Pixels?  Percentages?